Workflows/GIT/Clean changes.md

```diff
@@ -0,0 +1,131 @@
+After a while you'll get notifications about comments on your pull requests from your reviewers. Often times, you'll need to change something before the branch is ready.
+
+## Solving change requests
+Change requests should be applied in separate _fixup_ or _squash_ commits. Rebaseing the branch during an ongoing review is not appreciated unless there is a good reason for it, like pulling in some new and necessary changes from `master`, because it makes harder for the reviewers to know what the new changes are and what they already reviewed.
```

Comment:
```suggestion
Change requests should be applied in separate _fixup_ or _squash_ commits. Rebasing the branch during an ongoing review is not appreciated unless there is a good reason for it, like pulling in some new and necessary changes from `master`, because it makes it harder for the reviewers to know what the new changes are and what they have already reviewed.
```